### PR TITLE
Refactor Orchestrator

### PR DESCRIPTION
**Pull request by Timothylock**
Refactors the orchestrator and fixed some small bugs. 

*What this PR does not do* 
- Complete the full flow. This is merely a cleanup/refactoring PR.

### CODE REVIEW

File: src/task1/gui_broken_panel_filter/gui_sorter_working.py [hunk 1]
```diff
@@ -66,9 +66,7 @@ def getScaledDims(xdim, ydim):
 # Function called when we get to the end of the list of images to go through.
 def end_program(final_data):
 	with open(json_out_name, 'w') as outfile:
-		outfile.write("{\"Positive\": ")
-		json.dump({"Positive": final_data}, outfile)
-		outfile.write("}")
+		json.dump({"positive": final_data}, outfile)
```
Comment (Timothylock) [inline src/task1/gui_broken_panel_filter/gui_sorter_working.py]: the output was not only incorrect, but had `{"positive":{"positive": ... `

File: src/task1/map/map_coordinates.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
-	topleft: {
```
Comment (Timothylock) [inline src/task1/map/map_coordinates.json:2]: invalid json

File: src/task1/orchestrator/orchestrator.py [hunk 3]
```diff
@@ -171,8 +172,12 @@ def mainBGloop(self):
             self.plotButton.setEnabled(True)  # Plot Button unlock
             updateTextCompletedGreen(self.manualFilterLabel)
 
+        if True and not self.plotButton.isEnabled():  # DEBUG
```
Comment (Timothylock) [inline src/task1/orchestrator/orchestrator.py]: TODO: REMOVE THIS

File: src/task1/filter-module/detect.py [hunk 4]
```diff
@@ -40,14 +40,17 @@ def process_img(file_name):
...
-        for (i, c) in enumerate(cnts):
-            ((cX, cY), radius) = cv2.minEnclosingCircle(c)
-            cv2.circle(image, (int(cX), int(cY)), int(radius),
-                       (0, 0, 255), 2)
-        return image
-    return None
+
+    return image
+
+    # if cnts is not None and len(cnts) > 0:
```
Comment (Timothylock) [inline src/task1/filter-module/detect.py]: @xiejxie Still needed?

**Comment by Timothylock** [comment]
@derhabdu Added you because this PR contrary to what the title says actually calls your plotter

File: src/task1/orchestrator/mainWindowLayout.py [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+    btn.clicked.connect(window.exitProgram)
+    btn.resize(200, 50)
+    btn.move(730, 490)
+
+    # Step 1 Label
+    bl = QtWidgets.QLabel(window)
+    bl.setText("Step 1 -  select the source image folders")
+    bl.resize(600, 50)
+    bl.move(25, 20)
+    bl.setStyleSheet('color: darkblue')
```
Comment (xiejxie) [inline src/task1/orchestrator/mainWindowLayout.py:22]: fancy

File: src/task1/filter-module/detect.py (hunk 4 above)
Comment (xiejxie) [reply src/task1/filter-module/detect.py]: yep that's fine

File: src/task1/orchestrator/mainWindowLayout.py [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+    """
+    Draws the home layout
+
+    :param window: The QT session
+    :return:
+    """
+    # Module 1 Button
+    window.autofilterButton = QtWidgets.QPushButton("Module 1 - Auto Filter", window)
+    window.autofilterButton.clicked.connect(window.launchAutoFilterModule)
+    window.autofilterButton.move(25, 250)
```
Comment (xiejxie) [inline src/task1/orchestrator/mainWindowLayout.py:53]: Does pyqt have layout classes?

**Review by xiejxie — APPROVED** [review]
beautiful 👌✨
(Needs more snake case though and less absolute positioning)

File: src/task1/orchestrator/mainWindowLayout.py (hunk 6 above)
Comment (Timothylock) [reply src/task1/orchestrator/mainWindowLayout.py:53]: It does but we are way too deep into this to refactor :(

**Comment by Timothylock** [comment]
Thanks for the review @xiejxie , the absolute position is ~fixed~better improved in https://github.com/UTAT-AeRo/vision_201819/pull/33/commits/bd17a39e8632d45df77098046cd5feccabf990f7

**Pull request merged**